ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/Monitor.hs

```diff
@@ -624,59 +649,54 @@ monitorLedgerStateJudgement PeerSelectionActions{ readLedgerStateJudgement }
                                                    establishedPeers,
                                                    inProgressPromoteCold,
                                                    inProgressPromoteWarm,
-                                                   ledgerStateJudgement
+                                                   ledgerStateJudgement,
+                                                   useGenesisFlag
                                                  }
-  | isBootstrapPeersEnabled bootstrapPeersFlag =
+  | isBootstrapPeersEnabled bootstrapPeersFlag || useGenesisFlag =
```

Comment:
I think it will be better if we keep the two cases separate.  `bootstrap` peers are only until Genesis is deployed.